diagnostic.md

```diff
@@ -6,49 +6,56 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The back end is non user facing data and processes to support what goes on on the front end.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+The Model is used by the controller to carry out tasks
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+The model carries out the crud actions with commands from the controller and data from the database
 ```
 List at least 5 standard actions that C.R.U.D corresponds to?
 
 ```bash
-// your response here
+create index show read update destroy
 ```
 
 A user action fires a `PATCH` request for `pets/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list,
 please include information on dynamic segments, the params hash and seralizers).
 
 ```bash
-// your response here
+1. user send patch req
+2. the serializer checks for an available route
+3. the route is followed to the controller
+4. the controller then sends the command to the model
+5. the model looks at the pets table and gets the pet with ID #1
+6. that data is brought back to the controller
+7. the controller hands that data directly back to the client
 ```
 
 What is the command to scaffold a `medicalRecords` join table which holds
 refrences to a `pets` and a `vets` table?
 
 ```bash
-// your response here
+rails g scaffold medicalRecords pet_id:integer vet_id:integer
```

Comment:
nope, that's not how you scaffold references.  Should have used the repo we used in class, has this answer on it.